third_party/move/move-model/src/ast.rs

```diff
@@ -512,6 +526,34 @@ pub enum AddressSpecifier {
     Call(QualifiedInstId<FunId>, Symbol),
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Copy, Hash, Default)]
+pub enum LambdaCaptureKind {
+    /// No modifier (e.g., inlining)
+    #[default]
+    Default,
+    /// Copy
```

Comment:
This is kind of legacy from all the discussions, though we still might want to consider `copy`, to allow users to ask for an error if something needs to be moved.

Rust is more flexible in its use of references so it can more easily treat everything as a reference.  We did eventually decide to try to follow Rust, but originally I was thinking we might want to have `move` the default, with `borrow` an option.  I can clean it up now, I suppose.